lib/clients/websocket.js

```diff
@@ -24,15 +24,16 @@ class WebsocketClient extends EventEmitter {
         'Invalid or incomplete authentication credentials. You should either provide all of the secret, key and passphrase fields, or leave auth null'
       );
     }
-    this.channels = channels;
     this.auth = auth || {};
-    this.heartbeat = heartbeat;
+    this.channels = channels || ['full'];
+    if (!this.channels.includes('heartbeat')) {
+      this.channels.push('heartbeat');
```

Comment:
Heartbeats are pretty noisy, so there should be a way to disable them to save bandwidth.

Reply:
> Heartbeats are pretty noisy...

🤔 Not compared the `full` or `ticker` or `matches` or `level2` channel.  The only channel that is less noisy than `heartbeat` is the `user` channel.

`heartbeat` is mild compared to anything else, and it comes with other very useful benefits (keeps the connection open, helps address lost sequences, etc).

Also, the longer-term intent of this automatic `heartbeat` addition is to eventually allow the `WebsocketClient` to automatically monitor the heartbeat, reconnect as necessary, and potentially fire an event when a missed sequence is recognized. i.e. The `heartbeat` does not need to necessarily propagate to the caller.

(I'll also rebase after the new conflicts.)